### ISSUE TITLE
Handle erb comments

### ISSUE THREAD

**Issue by nevans** [nebulab/erb-formatter#21]
My code base had a few places where html had been temporarily commented out using `<%# etc etc %>`.  I don't want the comments to be formatted (other than perhaps indentation if they are multiline).  But erb-format misinterpreted the comments and seemed to treat the commented tags as real tags.

**Comment by elia** [comment]
I agree… if you have time to send that PR would be awesome, the starting point would be to adjust the expected output for multiline comments in https://github.com/nebulab/erb-formatter/blob/main/test/fixtures/comments.html.expected.erb
